src/file/DAL/fileRepository.ts

```diff
@@ -94,7 +101,15 @@ const createFileRepo = (dataSource: DataSource) => {
       try {
         return await this.manager.connection.transaction(isolationLevel, async (transactionalEntityManager) => {
           let completedSyncIds: string[] = [];
-          const completedFilesResult = await updateFileAsCompleted(fileId, schema, transactionalEntityManager);
+          const completedFilesResult: ReturningResult<ReturningId> = await (
```

Comment:
Hey Omer, Thanks for stopping by. 
Because we needed to use jest.spyon on `updateFileAsCompleted`, We had to export it. For some reason, We had to add `exports.` before `updateFileAsCompleted` because the test failed. without the exports, `jest.spyon`won't work and the test will fail.